modules/tensor_mechanics/test/src/materials/SS316HLAROMANCEStressUpdateTest.C

```diff
@@ -78,8 +89,9 @@ SS316LAROMData::getInputLimits() const
           {800.0160634, 949.9873586}};
 }
 
+template <ComputeStage compute_stage>
 std::vector<std::vector<Real>>
-SS316LAROMData::getCoefs() const
+SS316HLAROMANCEStressUpdateTest<compute_stage>::getCoefs()
```

Comment:
@bwspenc is this the best way to handle these 3072 coefficients?  Seems like some sort of reference might be better, sort of like what @tophmatthews had with the user object..would you explain the design decision to me so that I can learn from this change?

Reply:
@sapitts I think this is the same as how it was done with the UserObject, isn't it?
I think the code is OK. The data gets stored in `_coefs`, and once it's there, that's the only copy of it. I could be wrong here, but I think it's creating a local copy of the object that stores the data, and then copying that to _coefs, so that's a little sub-optimal, but it just happens once at the beginning of the run.  I guess putting a bunch of numbers in the code like this is going to increase the size of the executable too, but the only way around that is to store that data in a separate file. It's probably not enough data to get too worked up about in this case.

Reply:
Yeah, same way as the user object. It is loaded once. The placement in a function allows it to be overriden in inherited classes. In addition, it's cleaner than passing in an empty vector due to size intializations, etc.

Reply:
Thank you both for the explanation!